test/cmactest.c

```diff
@@ -333,7 +333,7 @@ static char *pt(unsigned char *md, unsigned int len)
     static char buf[80];
 
     for (i = 0; i < len; i++)
-        sprintf(&(buf[i * 2]), "%02x", md[i]);
+        BIO_snprintf(&(buf[i * 2]), sizeof(buf), "%02x", md[i]);
```

Comment:
`sizeof(buf)` is wrong, since we are writing to `buf + i * 2`. Either `sizeof(buf) - i * 2` or `2` would be better, but really we should check `len` here (but that's probably out of scope for this PR)